src/sdg/core/constants/product.py

```diff
@@ -15,3 +15,15 @@ def get_available_languages(cls):
 class DoelgroepChoices(DjangoChoices):
     burger = ChoiceItem("eu-burger", _("EU Burger"))
     bedrijf = ChoiceItem("eu-bedrijf", _("EU Bedrijf"))
+
+
+class ProductStatus(DjangoChoices):
+    NEW = ChoiceItem("nieuw", _("Nieuw"))
+    MONITOR = ChoiceItem("gereed-voor-beheer", _("Gereed voor beheer"))
+    PUBLICATE = ChoiceItem("gereed-voor-publicatie", _("Gereed voor publicatie"))
+    EXPIRED = ChoiceItem("vervallen-in-de-upl", _("Vervallen in de UPL"))
+    EOL = ChoiceItem("eol", _("End of life"))
+    DELETED = ChoiceItem("verwijdert", _("Verwijdert"))
+    MISSING = ChoiceItem(
+        "geen-product-gevonden", _("Geen referentie producten gevonden")
+    )
```

Comment:
I try to keep as much of the data layer in English, so lets use English constants that match the enum and Dutch translations.

```suggestion
    NEW = ChoiceItem("new", _("Nieuw"))
    READY_FOR_ADMIN = ChoiceItem("ready_admin", _("Gereed voor beheer"))
    READY_FOR_PUBLICATION = ChoiceItem("ready_publication", _("Gereed voor publicatie"))
    EXPIRED = ChoiceItem("expired_in_upl", _("Vervallen in de UPL"))
    EOL = ChoiceItem("eol", _("Wordt binnenkort verwijderd"))
    DELETED = ChoiceItem("deleted", _("Verwijderd"))
    MISSING = ChoiceItem(
        "missing", _("Geen referentie producten gevonden")
    )
```